src/components/browse/datasetlist/datasetheader/datasetdetails/datasetDetails.tsx

```diff
@@ -126,22 +136,83 @@ const DatasetDetails = (props: dataSetDetailsProps) => {
-                Metadata Catalogue. Please could you reply to me as soon as you
-                are able to discuss my proposed project? Thank you.
-                <br />
-                <br />
-                Kind regards
-              </p>
+              <Row>
+                <Col lg={11} md={11} sm={11} xl={11} xs={11} xxl={11}>
+                  To: {getEmailId()}
+                </Col>
+                <Col lg={1} md={1} sm={1} xl={1} xs={1} xxl={1}>
+                  <OverlayTrigger
+                    placement="right"
+                    delay={{ show: 250, hide: 400 }}
+                    overlay={renderTooltip("email ID")}
+                  >
+                    <CopyToClipboard text={copyEmail}>
+                      <Button id={"email ID"} variant="outline-dark">
+                        <FontAwesomeIcon
+                          icon={faCopy}
+                        />
+                      </Button>
+                    </CopyToClipboard>
+                  </OverlayTrigger>
+                </Col>
+              </Row>
+              <br />
+              <Row>
+                <Col lg={11} md={11} sm={11} xl={11} xs={11} xxl={11}>
+                  Subject: Request access for dataset {props.hit.content.accession}
+                </Col>
+                <Col lg={1} md={1} sm={1} xl={1} xs={1} xxl={1}>
+                  <OverlayTrigger
+                    placement="right"
+                    delay={{ show: 250, hide: 400 }}
+                    overlay={renderTooltip("subject")}
+                  >
+                    <CopyToClipboard text={"Request access for dataset " + props.hit.content.accession}>
+                      <Button id={"subject"} variant="outline-dark">
+                        <FontAwesomeIcon
+                          icon={faCopy}
+                        />
+                      </Button>
+                    </CopyToClipboard>
+                  </OverlayTrigger>
+                </Col>
+              </Row>
+              <br />
+              <Row>
+                <Col lg={11} md={11} sm={11} xl={11} xs={11} xxl={11}>
+                  Dear DAC team,
+                  <br />
+                  <br />I am interested in accessing the Dataset{" "}
+                  {props.hit.content.accession}, which is listed in the GHGA
+                  Metadata Catalogue. Please could you reply to me as soon as you
+                  are able to discuss my proposed project? Thank you.
+                  <br />
+                  <br />
+                  Kind regards
+                </Col>
+                <Col lg={1} md={1} sm={1} xl={1} xs={1} xxl={1}>
+                  <OverlayTrigger
+                    placement="right"
+                    delay={{ show: 250, hide: 400 }}
+                    overlay={renderTooltip("email body")}
+                  >
+                    <CopyToClipboard text={`Dear DAC team,
```

Comment:
Could we refactor this as a variable so we don't have to repeat text? Easy source of mistakes.

Reply:
This cannot be a single variable since, the string cannot process JSX elements like <br/>. I am not sure how to handle the new line in these two cases, any suggestion would be welcome.

Reply:
You do a string replace when rendering the html so it prints out <br/> instead of 2 newlines (\n\n)?

Reply:
This will be a new task.